### DISCUSSION TITLE
Add additional data to stats events

### DISCUSSION THREAD

**Discussion by neverything** [spatie/laravel-stats#11] [Ideas]
For our purposes we would like to add additional data to each stats event. Is that something that would be considered useful or should we simply use the activitylog package for this purpose instead?

Example: We have demo users on our app and would like to store additional info from their signup with the stats like region and school level. After 24h we automatically delete the demo user accounts from our database, but we add a stats event when they get signed up.

**Comment by freekmurze** [comment]
I think for your use case using our activity log package will be better.

**Reply by neverything** [reply]
Thanks for your reply
